packages/inspire/index.js

```diff
@@ -4,7 +4,7 @@
 // at the moment it still is the top level entry point, but this might change.
 import "@babel/polyfill";
 import injectTapEventPlugin from "react-tap-event-plugin";
-
+import isInBrowser from "is-in-browser";
```

Comment:
Same as above: replace with the updated ~/tools version. I think the inBrowser is importable via
```
import { inBrowser } from "~/tools";
```